src/client/pages/OfferNew/utils.ts

```diff
@@ -297,6 +297,10 @@ export const isNoDiscount = (campaigns: Campaign[]) =>
     campaigns[0].incentive.__typename === 'NoDiscount') ||
   false
 
+export const isNorwegianBundle = (offerData: OfferData) => {
+  return (isBundle(offerData) && isNorwegian(offerData)) || false
+}
```

Comment:
You could skip the explicit false value here I think since it would return false if any of the checks fails :) 
```suggestion
export const isNorwegianBundle = (offerData: OfferData): boolean =>
  isBundle(offerData) && isNorwegian(offerData)
```